api/proto/churner/churner.proto

```diff
@@ -41,6 +45,16 @@ message ChurnReply {
 	// The signature signed by the Churner.
 	SignatureWithSaltAndExpiry signature_with_salt_and_expiry = 1;
 	// A list of existing operators that get churned out.
+	// This list will contain the target operators to be churned out for all quorums specified
+	// in the ChurnRequest even if some quorums may not have any churned out operators.
+	// It is smart contract's responsibility to determine whether it needs to churn out
+	// these target operators based on whether the quorums have available space.
+	//
+	// For example, if the ChurnRequest specifies quorums 0 and 1 where quorum 0 is full
+	// and quorum 1 has available space, the ChurnReply will contain the operators to be
+	// churned out for both quorums 0 and 1 (operators with lowest stake). However,
```

Comment:
When there is no operator churned for a quorum, can you further make it clear what's included in OperatorToChurn (may comment right there), e.g. what are "operator" and "pubkey" field (I think they are just unset/undefined)?